ansible/playbooks/nagios/roles/Nagios_Config/tasks/run_py_script.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Run python script
+  script: /ansible/playbooks/nagios/roles/Nagios_Config/scripts/nagios_yaml.py
```

Comment:
According to https://docs.ansible.com/ansible/latest/collections/ansible/builtin/script_module.html
The script module will default to running it as a shell script. To run it as a python script use something like this
```
- name: Run a script using an executable in a system path
  ansible.builtin.script: /some/local/script.py
  args:
    executable: python3
```
(Found in the linked docs)

Reply:
oh yeah , refactoring with the new docs